src/pps/pps.proto

```diff
@@ -502,9 +485,10 @@ message ListJobRequest {
   string jqFilter = 6;
 }
 
-message FlushJobRequest {
-  repeated pfs_v2.Commit commits = 1;
-  repeated Pipeline to_pipelines = 2;
+// Streams active jobs until canceled
```

Comment:
We should probably get the naming down for terminal vs non-terminal jobs. Open / active seem fine for the non-terminal jobs.

Reply:
I think `active` sounds ok, but it implies that something is currently happening to it so I don't think it captures 'enqueued' jobs that are just waiting for parallelism or inputs.  'Open' kind of gets it, but maybe not as clear to other people?

Reply:
It doesn't seem like a bad idea to have consistent terminology between commits and jobs, particularly because they are mapped 1:1. So open commit and open job seem like good terms.